src/main/java/com/synopsys/integration/alert/provider/blackduck/collector/BlackDuckVulnerabilityCollector.java

```diff
@@ -183,6 +183,7 @@ private LinkableItem getSeverity(final String vulnerabilityUrl) {
         }
 
         severityItem.setSummarizable(true);
+        severityItem.setPartOfKey(true);
```

Comment:
Now that the `LinkableItem` name is the same for all vulnerabilities (rather than "High Vulnerabilities" etc) this is necessary. I does not appear to have an impact on other format types.